app/graph/types/inclusions/annotation_behaviors.rb

```diff
@@ -53,14 +53,6 @@ def assignments
         argument :annotation_type, GraphQL::Types::String, required: true, camelize: false
       end
 
-      def annotations(annotation_type:)
-        ::Annotation.where(
-          annotation_type: annotation_type,
-          annotated_type: ["Annotation", object.annotation_type.camelize],
-          annotated_id: object.id
-        )
-      end
```

Comment:
@jayjay-w I quickly `grep`'d by `annotations(` in Check Web and looks like this GraphQL field is still used in a few places, so please keep it and if coverage is affected, we can fix in a follow-up ticket.

Reply:
OK, Caio, I am keeping it.